playground/frontend/lib/modules/editor/components/share_dropdown/share_tab_body.dart

```diff
@@ -0,0 +1,91 @@
+// ignore_for_file: leading_newlines_in_multiline_strings
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:playground/constants/sizes.dart';
+import 'package:playground/modules/editor/components/share_dropdown/link_text_field.dart';
+import 'package:playground/pages/playground/states/examples_state.dart';
+import 'package:playground/pages/playground/states/playground_state.dart';
+import 'package:provider/provider.dart';
+
+import 'link_text_field.dart';
+
+class ShareTabBody extends StatefulWidget {
+  final String descriptionText;
+  final bool isCopyLinkTab;
```

Comment:
What if we add a third tab? A boolean should not be used for this.

Every time we find ourselves passing a flag to a widget that is later branches the result, we should really do one of the following to avoid such `if`s:
- Pass a child widget instead of the flag.
- Make 2 different widgets.

I think a child will do here.